plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/MigrateKVMAsync.java

```diff
@@ -36,6 +36,36 @@
     private boolean migrateStorage;
     private boolean autoConvergence;
 
+    /**
+     * Do not pause the domain during migration. The domain's memory will be transferred to the destination host while the domain is running. The migration may never converge if the domain is changing its memory faster then it can be transferred. The domain can be manually paused anytime during migration using virDomainSuspend.
+     * @value 1
+     * @see Libvirt <a href="https://libvirt.org/html/libvirt-libvirt-domain.html#virDomainMigrateFlags">virDomainMigrateFlags</a> documentation
+     */
+    private static final long VIR_MIGRATE_LIVE = 1l;
```

Comment:
The variable (`VIR_MIGRATE_LIVE`) is a primitive long; the JVM would make the Long into a primitive long via the auto unboxing process. I don't think that making it Long would make much a difference.
Are you talking about this recomendation?
> The suffix L is preferred, because the letter l (ell) is often hard to distinguish from the digit 1 (one).

I don't think that it would improve much the readability of the code here.

Reply:
@GabrielBrascher to people with weak eyes and misleading fonts configured `l` might look like a `1`. Please use `L`, even the jdk8 docs advise to use `L` than `l`.

Reply:
Will update it soon, thanks!

Reply:
Done!